forge/forge/llm/providers/schema.py

```diff
@@ -204,8 +204,7 @@ class ModelProviderCredentials(ProviderCredentials):
     api_version: SecretStr | None = UserConfigurable(default=None)
     deployment_id: SecretStr | None = UserConfigurable(default=None)
 
-    class Config(ProviderCredentials.Config):
-        extra = "ignore"
+    model_config = ConfigDict(extra="ignore")
```

Comment:
On `strict` mode, the type checker complains that "ignore" is not a valid value:
```suggestion
    model_config = ConfigDict(extra=Extra.ignore)
```

Reply:
Doesn't complain for me, and when I change it to `Extra.*`:

> Extra is deprecated. Use literal values instead (e.g. `extra='allow'`)